src/app/App.js

```diff
@@ -10,40 +10,48 @@ export const App = ({options}) => {
    showStartingPage();
 
    //add event listener to the help button 
+   const leader = document.querySelector('#leaderboardScreen');
+   const styleL = getComputedStyle(leader);
    document.querySelector('#helpOption').addEventListener('click', () => {
-      showAPopUpScreen(document.querySelector('#helpScreen'), 'initial');
+      if(styleL.display=='none'||leader.style.display == 'none')
+         showAPopUpScreen(document.querySelector('#helpScreen'), 'initial');
    });
 
    //add event listener to the select mode menu button 
-   const help=document.querySelector('#helpScreen')
-   const style = getComputedStyle(help);
+   const help = document.querySelector('#helpScreen');
+   const styleH = getComputedStyle(help);
    document.querySelector('#whoIsThatPokemonOption').addEventListener('click',()=>{
-      if(style.display=='none'||help.style.display == 'none')
+      if((styleH.display=='none'||help.style.display == 'none') && (styleL.display=='none'||leader.style.display == 'none')){
```

Comment:
isn't `styleH.display` the same as `help.style.display`? Or is there a difference?
I am asking because I don't know, but if it is the same - then it looks like code could be simplified by using only one of those

Reply:
Yes, it is a difference, because when I use one of this, first I had to click Help button or Leader button, and only then other button worked properly. But, I don't know why, I didn't think that I could use this second command alone. So I corrected it and now is less code and everythink work properly.